Pod/Classes/OAStackView.h

```diff
@@ -84,7 +84,7 @@ typedef NS_ENUM(NSInteger, OAStackViewAlignment) {
 NS_ASSUME_NONNULL_BEGIN
 @interface OAStackView : UIView
 
-@property(nonatomic,readonly,copy) NSArray *arrangedSubviews;
+@property(nonatomic,readonly,copy) NSMutableArray *arrangedSubviews;
```

Comment:
Looks good except the different API. As Grubas7 said I'd keep a mutable version internally (with a different property name) and override the arrangedSubviews getter to return a (non mutable) copy of the mutable array. This will allow you to keep the arrangedSubviews property in the header as an NSArray